AirCasting/Extensions/Date+Miliseconds.swift

```diff
@@ -24,4 +24,10 @@ extension Date {
         dateFormatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
         return dateFormatter.date(from: stringDate)!
     }
+
+    var roundedToSecond: Date {
```

Comment:
After thinking about this for a bit I think if we just need to round seconds we can do it in one line:
```Swift
Date(timeIntervalSinceReferenceDate: timeIntervalSinceReferenceDate.round())
```
(idk if naming is correct, was typing from memory)